src/Http/Client/FormDataPart.php

```diff
@@ -185,6 +194,16 @@ public function value()
         return $this->_value;
     }
 
+    /**
+     * Get the charset.
+     *
+     * @return string
+     */
+    public function charset()
```

Comment:
What's the use case for this public method?

Reply:
None. Got removed.